src/client/binary_client.cpp

```diff
@@ -375,7 +375,7 @@ namespace
       request.MethodsToCall = methodsToCall;
       const CallResponse response = Send<CallResponse>(request);
       if (Debug) {std::cout << "binary_clinent | Call <--" << std::endl;}
-      return response.Results;
+      return response.Parameters.Results;
```

Comment:
In the python file it is possible to specify if response is to be split in 2 structs or not. I could not decide on which is best. So you can decide if this Parameters stuct is necessary or not. It looks like it migth not be necessary here....